cmd/kubeadm/app/phases/addons/proxy/manifests.go

```diff
@@ -175,6 +175,20 @@ spec:
         - mountPath: /run/xtables.lock
           name: xtables-lock
           readOnly: false
+	  initContainers:
```

Comment:
Does this spacing align with the containers line?

Reply:
No....
Thanks for pointing out.